packages/optimizer/lib/transformers/ApplyLayout.js

```diff
@@ -79,6 +80,10 @@ function apply(layout, width, height, node) {
     case 'container':
       // Do nothing here.
       break;
+    case 'fluid':
+      styles += 'height:0;';
```

Comment:
Nit: we had `width:100%;height:0px;` in https://github.com/ampproject/amphtml/pull/34560/files#diff-ec6474c38021deb5cc02df92e38f7ae44a0bbda93384581872671a9c1c882d6fR292

Reply:
Added in d9876e2, but `0` seems better than `0px`